### DISCUSSION TITLE
Reset Skill Points

### DISCUSSION THREAD

**Discussion by xSubZero** [sunflower-land/sunflower-land#2028] [Ideas]
Hi everyone!

I come through this message to suggest with a certain urgency the reset of skill points, while leveling our bumpkin to level 30 and using the points on many skills, it didn't please some of us for several reasons, low skill activation (not previously disclosed), don't agregate too much to the person style of play, and others.

I suggest that the reset would be for a skill tree and not for all of them at once, for example, I have complete my Animal skill tree, using the reset, all those points invested in there would return so I can use them in Crops, Trees, Rocks or other available Skill Point Trees.

Price for reset skill point tree: 50 SFL

If it is not possible to reset the whole skill tree, we could reset the skills individually. 

Price for reset skill individually point:  10 SFL

This would help all players who are currently unhappy with their skills and we would have a burn on the token.

**Comment by MAISAbr** [comment]
Boa Sub!!!

**Reply by rasheed19871** [reply]
i like the reset but the cost is very high

**Comment by diasfigor** [comment]
Great idea, with this we can have several ways to grow in the game, opens a range for various game strategies.

**Comment by malkaron** [comment]
I love the possibility to reset only 1 element of the skill tree!

**Comment by future2back** [comment]
Honestly, I don't think it's fair to release the skill reset.

**Reply by brian-man** [reply]
> Honestly, I don't think it's fair to release the skill reset.

Interesting take.  How would it be **unfair** to do so?

**Reply by future2back** [reply]
Usually in RPG games, builds need to be taken very seriously, because it is what differentiates one player from the other. Hence the popup asking if you are sure you want to add your points there.

Do you think it's fair for someone to use the EXP skill to reach the maximum level then change it to another skill? I dont.

Why would you reset a skill?
- To take advantage.
- To remedy a bad gameplay.

Do you think it's fair?

**Reply by brian-man** [reply]
> Why would you reset a skill?
> * To take advantage.
> * To remedy a bad gameplay.
> 
> Do you think it's fair?

So when they add a new skill it should only be for new players?

You're missing a number of reasons:
- the skill was claimed during a testing period before the game was released and the player was doing the requested task during that period: testing
- the skill effect was changed by the devs after claim
- the skill effect was never implemented
- the skill is obsolete because the game changed dramatically
etc.

You seem to assume bad intent or player stupidity.  Neither is necessarily the case.

**Comment by future2back** [comment]
**So when they add a new skill it should only be for new players?**
    
     Friend, new levels are released to solve this.


**the skill was claimed during a testing period before the game was released and the player was doing the requested task during that period: testing**
    
    I was also before game was released and none skill using during test i want to change. 
    During the test period no one asked me to choose a certain skill.
    
**the skill effect was changed by the devs after claim**
    
    Which skill has changed the effect? If changed it was to a healthier game right?

**the skill effect was never implemented**
    
    This happened in the test period. A possible solution would be to return the points in exchange for the skill.


**the skill is obsolete because the game changed dramatically
    etc.**

    So all players have to be on meta always?

**Comment by DaewiTrain** [comment]
Hello, Sub! 👋
Great Idea! 
As of now, this proposal has my full support and consent.  @GranTY, please enter the paperwork, or rather, enter the codes.

**Comment by malkaron** [comment]
For me the reset should work like this, first 50 sfl, second 100, third 200... fourth 400.... fifth 800 + 20 gold, make the values very expensive to increase the sfl burnout well and at the same time be something serious.

**Comment by khronocross** [comment]
Honestly, I don't think it's fair to release the skill reset. In all kinds of games, we need to be careful with builds...

**Reply by caiofelipegq** [reply]
You certainly never played albion
